Client/Frontend/Browser/HomePanel/ntp-data.json

```diff
@@ -4,63 +4,63 @@
     "center": 1200,
     "credit": {
       "name": "Anders Jildén",
-      "url": "https://unsplash.com/@andersjilden?utm_source=unsplash&utm_medium=referral&utm_content=credit"
+      "url": "https://Unsplash.com/@andersjilden?utm_source=unsplash&utm_medium=referral&utm_content=credit"
```

Comment:
Capitalize first letter programatically. It will be easy to forget to capitalize it here when we add new json files